### PR TITLE
docs: reword publish to GitHub Pages documentation

### PR DESCRIPTION
**Pull request by M4rk9696**
## Motivation

I'm new to open-source contribution, really liked this project so I thought I'll start with some documentation(#853) to get used to the project.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Only the docs were changed. Run the website and check the formatting

### CODE REVIEW

File: docs/getting-started-publishing.md [hunk 1]
```diff
@@ -30,44 +30,53 @@ At this point, you can grab all of the files inside the `website/build` director
 
 ### Using GitHub Pages
 
-While choosing a web server or host is outside Docusaurus' scope, Docusaurus was designed to work really well with one of the most popular hosting solutions for open source projects: [GitHub Pages](https://pages.github.com/).
```
Comment (yangshun) [inline docs/getting-started-publishing.md]: Why is this removed?

File: docs/getting-started-publishing.md [hunk 2]
```diff
@@ -30,44 +30,53 @@ At this point, you can grab all of the files inside the `website/build` director
...
 | ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `organizationName` | The GitHub user or organization that owns the repository. In the case of Docusaurus, that would be the "facebook" GitHub organization.                                                   |
-| `projectName`      | The name of the GitHub repository for your project. For example, Docusaurus is hosted at https://github.com/facebook/docusaurus, so our project name in this case would be "docusaurus". |
+| `organizationName` | The GitHub user or organization that owns the repository. If you are the owner, then it is your GitHub username. In the case of Docusaurus, that would be the "_facebook_" GitHub organization.                                                   |
+| `projectName`      | The name of the GitHub repository for your project.  You may specify any other project of your choice. For example, Docusaurus is hosted at https://github.com/facebook/docusaurus, so our project name in this case would be "docusaurus".|
```
Comment (yangshun) [inline docs/getting-started-publishing.md]: > You may specify any other project of your choice.

This sounds odd and doesn't look right. It has to be your repo name.

File: docs/getting-started-publishing.md [hunk 3]
```diff
@@ -76,13 +85,20 @@ GIT_USER=<GIT_USER> \
...
+ You should now be able to load your website by visiting its GitHub Pages URL, which could be something along the lines of https://_username_.github.io/_projectName_, or a custom domain if you have set that up. For example, Docusaurus' own GitHub Pages URL is https://facebook.github.io/Docusaurus (it can also be accessed via https://docusaurus.io/), because it is served from the `gh-pages` branch of the https://github.com/facebook/docusaurus GitHub repository. We highly encourage reading through the [GitHub Pages documentation](https://pages.github.com) to learn more about how this hosting solution works.
 
-You can run the command above any time you update the docs and wish to deploy the changes to your site. Running the script manually may be fine for sites where the documentation rarely changes and it is not too much of an inconvenience to remember to manually deploy changes.
+ You can run the command above any time you update the docs and wish to deploy the changes to your site. Running the script manually may be fine for sites where the documentation rarely changes and it is not too much of an inconvenience to remember to manually deploy changes.
```
Comment (yangshun) [inline docs/getting-started-publishing.md]: Redundant leading whitespace here.

File: docs/getting-started-publishing.md [hunk 4]
```diff
@@ -76,13 +85,20 @@ GIT_USER=<GIT_USER> \
...
 
-However, you can automate the publishing process with continuous integration (CI).
+ However, you can automate the publishing process with continuous integration (CI).
```
Comment (yangshun) [inline docs/getting-started-publishing.md]: Redundant leading whitespace here.

**Comment by M4rk9696** [comment]
My aim was to improve the readability, so I changed the order of the content and made it as steps.

File: docs/getting-started-publishing.md (hunk 2 above)
Comment (M4rk9696) [reply docs/getting-started-publishing.md]: I thought that I'll mention that it was possible. Your right it sounds odd, I'll change it

**Comment by yangshun** [comment]
You're welcome @M4rk9696! Hope to see more contributions from you too! 😄
